zammad_py/api.py

```diff
@@ -276,6 +282,54 @@ def articles(self, id):
         )
         return self._raise_or_return_json(response)
 
+class Link(Resource):
```

Comment:
Can you rename it to TicketLink?
https://docs.zammad.org/en/latest/api/ticket/links.html Looks like link_object can be further extended to other types also.

Reply:
I can rename it, but that would break the convention you started of creating a class per api resource (the api endpoint being /links ).

If Zammad are extending the links endpoint to support other objects, the code could be used to also handle these objects, without having to rewrite / adding a "WhateverLink" Resource. 

I've updated the get-function to support that as well.

Reply:
ah okay. 👍. We can revisit this if they start extending then.